tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py310
+isolated_build = True
+
+
+[testenv]
+allowlist_externals = poetry
+commands_pre =
+    poetry install --only dev
```

Comment:
This really shoud install everything doesn't it? How will it know it install `pydantic` on a new, clean env?

Reply:
tox install the dependencies, I only need to explicitly install the dev dependencies.